integrations/server/test_covidcast.py

```diff
@@ -37,23 +37,22 @@ def _insert_placeholder_set_one(self):
 
   def _insert_placeholder_set_two(self):
     rows = [
-      CovidcastTestRow.make_default_row(geo_type='county', geo_value=str(i)*5, value=i*1., stderr=i*10., sample_size=i*100.)
+      CovidcastTestRow.make_default_row(geo_type='msa', geo_value=MSA[i-1], value=i*1., stderr=i*10., sample_size=i*100.)
       for i in [1, 2, 3]
     ] + [
-      # geo value intended to overlap with counties above
-      CovidcastTestRow.make_default_row(geo_type='msa', geo_value=str(i-3)*5, value=i*1., stderr=i*10., sample_size=i*100.)
+      CovidcastTestRow.make_default_row(geo_type='fips', geo_value=FIPS[i-4], value=i*1., stderr=i*10., sample_size=i*100.)
       for i in [4, 5, 6]
     ]
     self._insert_rows(rows)
     return rows
 
   def _insert_placeholder_set_three(self):
     rows = [
-      CovidcastTestRow.make_default_row(geo_type='county', geo_value='11111', time_value=2000_01_01+i, value=i*1., stderr=i*10., sample_size=i*100., issue=2000_01_03, lag=2-i)
+      CovidcastTestRow.make_default_row(geo_value=MSA[0], time_value=2000_01_01+i, value=i*1., stderr=i*10., sample_size=i*100., issue=2000_01_03, lag=2-i)
       for i in [1, 2, 3]
     ] + [
-      # time value intended to overlap with 11111 above, with disjoint geo values
-      CovidcastTestRow.make_default_row(geo_type='county', geo_value=str(i)*5, time_value=2000_01_01+i-3, value=i*1., stderr=i*10., sample_size=i*100., issue=2000_01_03, lag=5-i)
+      # time value intended to overlap with the time values above, with disjoint geo values
+      CovidcastTestRow.make_default_row(geo_value=str(i)*5, time_value=2000_01_01+i-3, value=i*1., stderr=i*10., sample_size=i*100., issue=2000_01_03, lag=5-i)
```

Comment:
is it your intention to make rows with invalid `geo_value`s here?  maybe you want to use some from `FIPS` instead?

Reply:
Oh sorry that I overlooked these comments, somehow it did not show on the conversation page. Yeah I think using some FIPS here would be good.
I think we don't see "invalid geo" because the make_default_row(base function [here](https://github.com/cmu-delphi/delphi-epidata/blob/61e545f1bb865a231e32d4b23c1d3961a0c617ed/src/acquisition/covidcast/database.py#L110)) does not go through the geo validator in the server maybe?

Reply:
`make_default_row()` doesnt check them, youre right.  it looks like whenever sample rows from `_insert_placeholder_set_three` are used, certain subsets of them are isolated and sometimes the `geo_value`s are changed before theyre passed to `request_based_on_row()` where they are actually checked for geo validation, and bad `geo_value`s were never sent in a request.